api/urls.py

```diff
@@ -10,5 +10,7 @@
     re_path('pipeline-image', views.pipeline_from_github_api_endpoint),
     re_path('ga-version', views.ga_version),
     re_path('branch/', views.branch_data, name='branch_data_view'),
-    re_path('test', views.test)
+    re_path('test', views.test),
+    re_path('fetch-github-data', views.fetch_github_data)
```

Comment:
`fetch-github-data` seems to be a bit vague. It might be give it a better name, which matches what we're doing, like `list-rpm-images`